src/resources/help/en_US/relnotes3.3.2.html

```diff
@@ -32,6 +32,7 @@
   <li>Fixed a bug with Pick where it incorrectly used the node origin.</li>
   <li>Fixed a bug where "atan2" did not show up in the list of expressions in the Expressions window.</li>
   <li>Fixed a bug with the Mili reader where variables that were only defined on a portion of the nodes weren't being placed on the correct nodes.</li>
+  <li>Fixed a bug with the python cli where it would give a poor error message when passing an integer instead of a string.</li>
```

Comment:
```suggestion
  <li>Fixed a bug with the python cli's TurnOffMaterials() method where it would give a poor error message when passing an integer instead of a string.</li>
```

Reply:
Actually, my apologies but since this commit is to `develop`, which is fine, it won't be in any 3.3.X release. It will be in the 3.4.0 release. So, remove this edit from 3.3.2 relnotes and instead, add it to 3.4 file there...if there is one. There may not be in which case, don't worry about it.